frontend/src/app/patients/PatientForm.tsx

```diff
@@ -297,8 +297,7 @@ const PatientForm = (props: Props) => {
             toast,
             <Alert
               type="success"
-              title={`${PATIENT_TERM_CAP} Record Saved`}
-              body="Information record has been updated."
+              title={`Your profile changes have been saved`}
```

Comment:
When a healthcare worker uses this, the "your" isn't right but the effect on one-time users is probably higher. Could we minimize the problem with a simple "Profile changes have been saved" perhaps?

Reply:
This alert is only for the patient view. The regular edit patient alert still shows the original message. Unless I'm missing something where this could go to the healthcare worker too?